extensions/eda/plugins/event_source/aws_cloudtrail.py

```diff
@@ -39,6 +6,63 @@
 from aiobotocore.session import get_session
 from botocore.client import BaseClient
 
+DOCUMENTATION = r"""
+---
+short_description: Receive events from an AWS CloudTrail
+description:
```

Comment:
Here we should mention that it is supported all the authentication methods supported by boto library and point to 
"https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html"